src/components/charts/bubble-chart/bubble-chart.js

```diff
@@ -25,10 +26,18 @@ class BubbleChart extends PureComponent {
     return bubble(root);
   };
 
-  getTooltipText = ({ tooltipContent, value, unit }) =>
-    tooltipContent && tooltipContent.length
-      ? tooltipContent.join('<br>')
-      : `${value} ${unit}`;
+  formatValue = (value, unit, { scale, suffix, format: f }) =>
+    `${format(f)(value * scale)}${suffix} ${unit}`;
+
+  getTooltipText = ({ tooltipContent, value, unit }, config) => {
+    if (tooltipContent && tooltipContent.length) {
+      const formattedContent = tooltipContent.map(
+        v => typeof v === 'number' ? this.formatValue(v, unit, config) : v
+      );
+      return formattedContent.join('<br>');
```

Comment:
I would prefer to use divs or p instead of br: https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br

Reply:
I did this according to the documentation of react-tooltip library:
https://www.npmjs.com/package/react-tooltip
![screenshot from 2018-11-12 12-20-11](https://user-images.githubusercontent.com/15097138/48347248-9a5abc80-e675-11e8-844a-37af3221bbcc.png)
At the end, in the browser there is no `<br>`, react-tooltip splits text lines into separate span elements (or div), so I see that this is only information for ReactTooltip component where split text into new lines and put this text into different logical blocks:
![screenshot from 2018-11-12 12-27-19](https://user-images.githubusercontent.com/15097138/48347504-56b48280-e676-11e8-9bff-9a84984d16eb.png)
So we can leave it like that?

About default styles of tooltip I can change it, no problem :)

Reply:
Cool, I didn't know that